src/components/SideBar/index.jsx

```diff
@@ -37,6 +51,11 @@ function SideBar(props) {
             <FavoriteBorderIcon className="icon" /> <span>Favourites</span>
           </div>
         </li>
+        <li onClick={handleOpenNotificationPopup}>
```

Comment:
instead of adding in the sidebar, add this in navbar, besides toggle button of row/column display